src/styles/_layout.scss

```diff
@@ -61,16 +52,38 @@
   }
 
   > main:not(.scrolling) > article > div {
-    /* fix: dirty overlapping of orange outline with neighboring rows */
-    margin: $engine_hover_outline_width 0;
+    /* use box-shadow to create collapsing border (@see https://stackoverflow.com/a/28807765) */
+
+    /* box-shadow:
+      2px 0 0 0 #eee,
+      0 2px 0 0 #eee,
+      2px 2px 0 0 #eee,
+      2px 0 0 0 #eee inset,
+      0 2px 0 0 #eee inset; */
+
+    padding-top: 2px;
 
     &:hover {
-      outline: $engine_hover_outline_color;
+      position: relative;
       z-index: 2;
+      box-shadow:
```

Comment:
can you extract this complex box-shadow construct to a mixin? then it could be used within lineup, too